client/components/summary/index.js

```diff
@@ -0,0 +1,96 @@
+import React, { PropTypes } from 'react';
+import find from 'lodash/find';
+
+const Summary = ( { overrideFields, formValues, layoutItems, summaryTemplate } ) => {
```

Comment:
This component name is also too broad. I have no idea what it does from the name nor the code. Please choose a more descriptive name.